app/src/main/java/nerd/tuxmobil/fahrplan/congress/details/SessionDetailsViewModel.kt

```diff
@@ -30,8 +31,8 @@ class SessionDetailsViewModel @JvmOverloads constructor(
         private val toFeedbackUrl: Session.(String) -> String = { urlTemplate ->
             FeedbackUrlComposer(this, urlTemplate).getFeedbackUrl()
         },
-        private val toPlainText: Session.() -> String = {
-            SimpleSessionFormat.format(this)
+        private val plainTextOf: (Session, ZoneId?) -> String = { session, timeZoneId ->
```

Comment:
Why not `Session.(ZoneId?) -> String`?

Reply:
Good point. Changed in 1f52d033eb0592c50aa2b707b274883bfdedf500.